pkg/util/fast_int_set.go

```diff
@@ -17,10 +17,11 @@ package util
 import (
 	"bytes"
 	"fmt"
+	"sort"
 )
 
-// FastIntSet keeps track of a set of integers. It is very fast when the values
-// are small. It is not thread-safe.
+// FastIntSet keeps track of a set of integers. It does not perform any
+// allocations when the values are small. It is not thread-safe.
 type FastIntSet struct {
 	smallVals uint64
 	largeVals map[uint32]struct{}
```

Comment:
Why? It's not allocated unless we actually need it. And there are usecases of FastIntSet where your solution is terrible (inserting a few large values, which we conceivably could do in other places where we use this structure, e.g. we use it for `ColumnIDs` in another context and there are no guarantees that IDs will always be small).